include/gz/math/SphericalCoordinates.hh

```diff
@@ -167,6 +240,18 @@ namespace gz
       /// \param[in] _type SurfaceType value.
       public: void SetSurface(const SurfaceType &_type);
 
+      /// \brief Set SurfaceType for planetary surface model with
+      /// custom ellipsoid properties.
+      /// \param[in] _type SurfaceType value.
+      /// \param[in] _axisEquatorial Equatorial axis of the surface.
+      /// \param[in] _axisPolar Polar axis of the surface.
+      /// \param[in] _flattening Falttening parameter of the surface.
+      public: void SetSurface(
+                  const SurfaceType &_type,
+                  const double _axisEquatorial,
+                  const double _axisPolar,
+                  const double _flattening);
```

Comment:
Shouldn't we have the `_radius` as a parameter as well?

Reply:
Done here : https://github.com/gazebosim/gz-math/pull/434/commits/765b8d9286ae44a856a61269e5c28bc159e3589b